examples/service-binding.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: servicecatalog.k8s.io/v1beta1
```

Comment:
For these examples would be good to include something about them in the README maybe?

Reply:
Also considering this is an example repo anyways, maybe we could rename this directory from `examples` to `manifests` or something better? :) WDYT?

Reply:
I had the same thought to do.  I'll merge this and do it in a follow-up.